GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -34,6 +35,11 @@ public override void ConfigureVisualStudio(string gitBinPath, ITracer tracer)
         public override bool TryGetGVFSHooksPathAndVersion(out string hooksPaths, out string hooksVersion, out string error)
         {
             hooksPaths = string.Empty;
+            string binPath = $"/usr/local/vfsforgit/{GVFSPlatform.Instance.Constants.GVFSHooksExecutableName}";

```

Comment:
Can we use `GetGVFSHooksPath` here?

Reply:
`GetGVFSHooksPath` was redundant. Rest of the code uses only `TryGetGVFSHooksPathAndVersion`. Removed it.